src/instance/minecraft/commands/SlayerCommand.ts

```diff
@@ -0,0 +1,113 @@
+import * as assert from 'node:assert'
+import { Slayer } from 'hypixel-api-reborn'
+import { ChatCommandContext, ChatCommandHandler } from '../common/ChatInterface'
+
+const Slayers: Record<string, string[]> = {
+  zombie: ['revenant', 'rev', 'zombie'],
+  spider: ['tarantula', 'tara', 'spider'],
+  wolf: ['sven', 'wolf'],
+  enderman: ['voidgloom', 'eman', 'enderman'],
+  blaze: ['inferno', 'demonlord', 'blaze'],
+  vampire: ['riftstalker', 'bloodfiend', 'vamp', 'vampire'],
+  overview: ['all', 'summary']
+}
+
+export default {
+  name: 'Secrets',
+  triggers: ['slayer', 'sl', 'slyr'],
+  description: "Returns a player's slayer levels",
+  example: `slayer eman %s`,
+  enabled: true,
+  handler: async function (context: ChatCommandContext): Promise<string> {
+    const givenSlayer = context.args[0] ?? 'overview'
+    const givenUsername = context.args[1] ?? context.username
+
+    const uuid = await context.clientInstance.app.mojangApi
+      .profileByUsername(givenUsername)
+      .then((mojangProfile) => mojangProfile.id)
+      .catch(() => {
+        /* return undefined */
+      })
+
+    if (uuid == undefined) {
+      return `${context.username}, Invalid username! (given: ${givenUsername})`
+    }
+
+    const hypixelProfile = await context.clientInstance.app.hypixelApi.getPlayer(uuid)
+    assert(hypixelProfile)
+
+    const slayers = await context.clientInstance.app.hypixelApi
```

Comment:
Change it to `slayersProfile`. `Slayers` already used. Taking advantage of case-sensitive variables might be confusing to readers.

Reply:
done